airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/oauth.py

```diff
@@ -214,10 +215,12 @@ def get_access_token(self) -> str:
             self.access_token = new_access_token
             self.set_refresh_token(new_refresh_token)
             self.set_token_expiry_date(new_token_expiry_date)
-            if self._message_repository:
+            # FIXME emit_configuration_as_airbyte_control_message as been deprecated in favor of package airbyte_cdk.sources.message
```

Comment:
are the messages emitted by the message repository not printed to stdout by the entrypoint?

Reply:
I think the issue is that we have defined a default value for `message_repository` to keep backward compatibility. If a message_repository is not provide, the behavior would change. As `NoopMessageRepository` is the default class for `message_repository`, we basically need to check "if default value, keep previous behavior". Changing this would mean breaking the config control message for all oauth sources that aren't declarative as only `ManifestDeclarativeSource` defines a message repository today. This can't be in the `AbstractSource` as it is abstract